nuntium/models.py

```diff
@@ -55,6 +55,13 @@ class WriteItInstance(models.Model):
         through='Membership')
     owner = models.ForeignKey(User, related_name="writeitinstances")
 
+    def add_person(self, person):
+        Membership.objects.get_or_create(writeitinstance=self, person=person)
+
+    @property
+    def persons_with_contacts(self):
+        return self.persons.filter(contact__isnull=False)
```

Comment:
Actually this is a deeper thing, a person could have contacts related to another instance and therefore would be appearing in this queryset too! thanks this was a bug